scripts/spe497-remove-tools-buckets.ts

```
@@ -0,0 +1,75 @@
+    requireEnv('SUPABASE_SERVICE_ROLE_KEY')
+  );
+
+  for (const bucket of BUCKETS) {
+    const { data: bucketInfo, error: headErr } = await supabase.storage.getBucket(bucket);
+    if (headErr) {
+      // Only a real not-found means "already removed". Anything else (network,
+      // auth, 5xx) must abort loudly — this script is the ONLY remaining
+      // deletion path for this data; a silent skip would report success while
+      // student-work objects survive.
+      if (/not.*found/i.test(headErr.message)) {
+        console.log(`[${bucket}] not found — already removed, skipping`);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)